modules/core/src/test/java/org/apache/ignite/internal/metric/CacheMetricsAddRemoveTest.java

```diff
@@ -161,6 +200,9 @@ private void createCache(@Nullable String dataRegionName, @Nullable String cache
         if (cacheName != null)
             ccfg.setName(cacheName);
 
+        if (groupName != null)
+            ccfg.setGroupName(groupName);
+
```

Comment:
I propose to set cache mode as well. It changes in parameters but does not actually apply to the cache.

Reply:
I think this class should be refactored. This is not related to the issue and should be done in a separate issue.